est.nf

```diff
@@ -231,6 +250,9 @@ workflow {
         sequence_id_files = fasta_import_files
     } else {
         sequence_id_files = get_sequence_ids()
+
+        // split up the sequence ID list into separate files so we can parallelize the
+        // process of retrieving sequences from the BLAST sequence database
```

Comment:
This is admittedly very pedantic but we are not _parallelizing_ so much as we are _making it parallelizable_. We should get in the habit of thinking that nextflow decides how to paralleize it, we just define the dependency graph. It would be more accurate to say that we split the file to enable parallel sequence retrieval